util/sdk/macros/index.mjs

```diff
@@ -354,8 +416,38 @@ const generateImports = json => {
   return imports
 }
 
-const generateInitialization = json => generateEventInitialization(json) + '\n' + generateProviderInitialization(json) + '\n' + generateDeprecatedInitialization(json)
+const generateInitialization = json => [generateEventInitialization(json), generateProviderInitialization(json), generateDeprecatedInitialization(json)].join('\n')
+
+const generateConfig = (json, pkg) => [generateHttpConfig(json, pkg)].join('\n')
+
+const generateHttpConfig = (json, pkg) => {
+  if (httpMethodsOrEmptyArray(json).length) {
+    const config =
+`Configuration.set('apiBaseUri', '${json.info["x-http-endpoint"]}')
+
+function _initialize(config, apis) {
+  Configuration.set(config)
+  Http.setEndpoint(Configuration.get('apiBaseUri'))
+  Http.onToken(apis.token)
+  Http.onAuthorize(apis.authorize)
+}
+
+export const initialize = window.__firebolt ? () => { throw new Error('Use Extensions.initialize() from \\'@firebolt-js/sdk\\' to initialize ${pkg.name}.') } : _initialize
```

Comment:
think I might need this walked through. Why would the initialize method throw an error if __firebolt is not undefined?

Reply:
This `export const initialize` is for non-Firebolt apps that need to use the SDK. They can import the initialize method and use the SDK w/out Firebolt. Not a high priority use case, but seemed like a good bonus to get teams to leverage our framework.

Firebolt get's access to the _real_ initialize method (not the exported one) via `window.__firebolt.registerExtensionSDK`.